qiskit/visualization/matplotlib.py

```diff
@@ -947,7 +963,7 @@ def _draw_ops(self, verbose=False):
                     self._line(qreg_b, qreg_t, lc=lc)
 
                 # cswap gate
-                elif op.name != 'swap' and base_name == 'swap':
+                elif op.name == 'cswap':
```

Comment:
This will not cover swap gates with more than one control, like `ccswap`, so should go back to original.

Reply:
Yeah. I also figured this problem. The current version also doesn't cover for cases with more than 1 controlling qubit. Since the original problem asked for colors change only, I modified the code to match it up with IQX reference. Should we creat a different issue for it?

Reply:
I understand why you did it, but we have to be careful we don't lose functionality. Adding a separate elif would be better. I left a comment on the original issue. I feel like we're working with an incomplete spec.

Reply:
The specifications are complete, see the comment below